modules/dpu-enable-dpu-hw-offloading-infra.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//CC-3 (alongside 4.10 dev preview)
+// * hardware_enablement/dpu-hardware-offload.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-hardware-offloading-infra_{context}"]
+= Configuring support for hardware offloading in the infrastructure cluster
+
+Configure support for hardware offloading in the infrastructure cluster by using this procedure.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following OVNKubeConfig custom resource (CR) with the poolName `dpu`, leaving the `kubeConfigFile` blank. Save the YAML in the `ovnkubeconfig.yaml` file:
```

Comment:
Not sure why here create the DpuClusterConfig CR and set kubeConfigFile blank. and then do update in `Final infrastructure cluster configuration`,  Can we move `Final infrastructure cluster configuration` part here?

Reply:
I cannot recall the reason exactly @bn222 might but I thought it was required to be staged this way. Maybe I picked it up incorrectly.

Reply:
The reason why we have this is that we advocate:

1. installing the control plane of the infra cluster
2. add BF workers + configure dpu operator (with kubeconfig set to blank)
3. Installing the control plan
4. adding workers + configure sriov operator
5. set the kubeconfig in DpuClusterConfig now that it's available

As you can see, there is a circular dependency issue here that be break by setting the kubeconfig to blank initially. 

An alternative approach would be to set up both control planes initially. In that case, we would have the kubeconfig of the tenant cluster while we are setting up DPU operator. It's possible, but we just don't it like that.

@kquinn1204 if we would write the docs in that way, would that make the docs significantly shorter? If yes, it would make sense to rewrite it like that.

Reply:
@bn222 I would need to look into that, though I would hope we can just QE the process as currently documented in this PR. I think that makes the most sense.